### PR TITLE
Adds the Memorize package

### PR DESCRIPTION
**Pull request by ckm2k1**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.

My package is a visual memory aid designed to help explore call stacks in code.

There are no packages like it in Package Control.

### CODE REVIEW

**Comment by kaste** [comment]
LGTM from me.  👍   You should improve your README though!  

Btw, I don't see it tightly coupled to the idea of call stacks, as it remembers and visualizes any file loc.  E.g.

image

Maybe there should be a way to re-order these bits, and remove some frames.  I also wish the stack-view would be visible while navigating around.  (Maybe an output panel at the bottom would be a good fit?)  But these are just ideas...

**Comment by ckm2k1** [comment]
Thanks @kaste! All of your suggestions are already on my todo list :)
A better README is coming shortly!

**Comment by kaste** [comment]
You restrict this to Mac/Linux, but the current version runs on Windows too.  (I'm on Windows.)  Maybe change that?

**Comment by ckm2k1** [comment]
> You restrict this to Mac/Linux, but the current version runs on Windows too. (I'm on Windows.) Maybe change that?

I didn't have a Windows machine to test this on so I chose to be conservative about claiming to support it, but given you ran it successfully, I'll tag it as supporting.

File: repository/m.json [hunk 1]
```diff
@@ -1295,6 +1295,18 @@
...
 		},
+		{
+			"name": "Memorize",
+			"details": "https://github.com/ckm2k1/sublime_memorize",
+			"labels": ["call stack", "memory aid"],
+			"releases": [
+				{
+					"sublime_text": ">=4180",
+					"tags": true,
+					"platforms": ["osx", "linux", "windows"]
```
Comment (braver) [inline repository/m.json:1306]: Since you now support all known platforms, just remove that line 😉

Comment (ckm2k1) [reply repository/m.json]: Done

**Comment by braver** [comment]
One last thing (sorry, should have mentioned that yesterday): if you add a main menu entry for the keybindings, that makes them easier to find.

**Comment by ckm2k1** [comment]
@braver - done, added a menu entry for keybindings

**Pull request merged**